components/src/components/BigTranscript.tsx

```diff
@@ -61,28 +62,29 @@ export const BigTranscript: React.FC = props => {
         const key = `${segment.contextId}/${segment.id}/${index}`
         return (
           <span key={key}>
-            <TransscriptItem entityType={w.entityType}>{w.word}</TransscriptItem>{' '}
+            <TransscriptItem word={w}>{w.word}</TransscriptItem>{' '}
           </span>
         )
       })}
     </BigTranscriptDiv>
   )
 }
 
-const TransscriptItem: React.FC<{ entityType: string | null }> = props => {
+const TransscriptItem: React.FC<{ word: ITaggedWord }> = props => {
   const [springProps] = useSpring(() => ({
     from: { opacity: 0 },
     to: { opacity: 1 },
     config: { tension: 500 },
   }))
 
   const entityProps = useSpring({
-    entityEffect: props.entityType !== null ? 1 : 0,
+    entityEffect: props.word.entityType !== null ? 1 : 0,
     config: { duration: 250 },
   })
 
   return (
-    <TransscriptItemDiv className={props.entityType !== null ? `Entity ${props.entityType}` : ''}>
+    // eslint-disable-next-line
+    <TransscriptItemDiv className={`${props.word.entityType ? 'Entity' : ''} ${props.word.isFinal ? 'Final' : ''} ${props.word.entityType || ''}`}>
```

Comment:
You can utilise `??` operator to make it a bit cleaner and pass liner checks.
```suggestion
    <TransscriptItemDiv className={`${props.word.entityType ?? 'Entity'} ${props.word.isFinal ? 'Final' : ''} ${props.word.entityType ?? ''}`}>
```

Reply:
Changed as suggested and removed disabled linting